easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -513,6 +513,11 @@ def __init__(self, *args, **kwargs):
         # Don't let pip connect to PYPI to check for a new version
         env.setvar('PIP_DISABLE_PIP_VERSION_CHECK', 'true')
 
+        # avoid that lib subdirs are appended to $*LIBRARY_PATH if they don't provide libraries
+        # typically, only lib/pythonX.Y/site-packages should be added to $PYTHONPATH (see make_module_extra)
+        self.module_load_environment.LD_LIBRARY_PATH.type = "PATH_WITH_TOP_FILES"
+        self.module_load_environment.LIBRARY_PATH.type = "PATH_WITH_TOP_FILES"
```

Comment:
We should use `ModEnvVarType.PATH_WITH_TOP_FILES` here, not some magic string value...

Reply:
fixed in 75f755c